content/SCALE/SCALEUIReference/Apps/InstallCustomAppScreens.md

```diff
@@ -0,0 +1,274 @@
+
+See the [Kubernetes documentation](https://kubernetes.io/docs/concepts/services-networking/) for more details on host networking. 
+You can create additional network interfaces for the container or give static IP addresses and routes to a new interface. 
+By default, containers use the DNS settings from the host system.
+You can change the DNS policy and define separate nameservers and search domains.
+See the Docker [DNS services documentation](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/) for more details.
+
+{{< trueimage src="/images/SCALE/23.10/InstallCustomAppNetworking.png" alt="Networking Settings" id="7: Networking Settings" >}}
+
+{{< expand "Settings Information" "v" >}}
+
+{{< trueimage src="/images/SCALE/23.10/InstallCustomAppNetworkingAddExternalInterfaces.png" alt="Networking Add External Interfaces" id="8: Networking Add External Interfaces" >}}
+
+{{< truetable >}}
+| Setting | Description |
+|---------|-------------|
+| **Add External Interfaces** | Click **Add** to displays the **Host Interface** and **IP Address Management** settings. |
+| **Host Interface** | Required. Select a host interface configured on your system from the dropdown list. |
+| **IPAM Type** | Required. Select an **IP Address Management** option from the dropdown list. Options are **Use DHCP** or **Use Static IP**. <br>**Use Static IP** adds two settings, **Static IP Address** and **Static Routes**. Click **Add** to the right of **Static IP Addresses** to display the **Static IP** fields to specify the IP address and CIDR value. <br>Click **Add** to the right of **Static Routes** to add the **Destination** and **Gateway** fields.  |
+{{< /truetable >}}
+
+{{< trueimage src="/images/SCALE/23.10/InstallCustomAppNetworkingDNSConfig.png" alt="Networking Add DNS Configuration" id="9: Networking Add DNS Configuration" >}}
+
+{{< truetable >}}
+| Setting | Description |
+|---------|-------------|
+| **DNS Policy** | Select the DNS policy option from the dropdown list. With the default behavior, the pod inherits the name resolution configuration from the node that the pods run on. With **None**, a pod can ignore DNS settings from the Kubernetes environment. Options are:<br> **Use Default DNS Policy where Pod inherits the name resolution configuration from the node**.<br> **Kubernetes internal DNS is prioritized and resolved first.** If the domain does not resolve with internal Kubernetes DNS, the DNS query forwards to the upstream nameserver inherited from the node, which is useful if the workload needs to access other services using Kubernetes internal DNS.<br> **For Pods running with hostNetwork and wanting to prioritize internal Kubernetes DNS should make use of this policy.**<br> **Ignore DNS settings from the Kubernetes cluster**. For more information on Kubernetes DNS policies see [Pod DNS Policy](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy). |
```

Comment:
Having this as a broken out list without bullets (because of the table) is a little confusing to read in the rendered site. 

Maybe something like


```suggestion
| **DNS Policy** | Select the DNS policy option from the dropdown list. With the default behavior, the pod inherits the name resolution configuration from the node that the pods run on. With **None**, a pod can ignore DNS settings from the Kubernetes environment. Options are:<br> **1. Use Default DNS Policy where Pod inherits the name resolution configuration from the node**.<br> **2. Kubernetes internal DNS is prioritized and resolved first.** If the domain does not resolve with internal Kubernetes DNS, the DNS query forwards to the upstream nameserver inherited from the node, which is useful if the workload needs to access other services using Kubernetes internal DNS.<br> **3. For Pods running with hostNetwork and wanting to prioritize internal Kubernetes DNS should make use of this policy.**<br> **4. Ignore DNS settings from the Kubernetes cluster**. For more information on Kubernetes DNS policies see [Pod DNS Policy](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy). |
```

Reply:
this is not a step procedure so it does not need and should not use an ordered list. Also, this dropdown list is a hot mess because of the way the devs entered the options for this list. I'm rejecting this suggestion. Until devs make UI changes there is not much we can do here.

Reply:
What about an unordered list using html tags then? Just using breaks and a jumble of left justified text, some bold and some not is not easy on the eyes.


```suggestion
| **DNS Policy** | Select the DNS policy option from the dropdown list. With the default behavior, the pod inherits the name resolution configuration from the node that the pods run on. With **None**, a pod can ignore DNS settings from the Kubernetes environment. Options are:<ul> <li>**Use Default DNS Policy where Pod inherits the name resolution configuration from the node**.<br> <li>**Kubernetes internal DNS is prioritized and resolved first.** If the domain does not resolve with internal Kubernetes DNS, the DNS query forwards to the upstream nameserver inherited from the node, which is useful if the workload needs to access other services using Kubernetes internal DNS.<br> <li>**For Pods running with hostNetwork and wanting to prioritize internal Kubernetes DNS should make use of this policy.**<br> <li>**Ignore DNS settings from the Kubernetes cluster**. For more information on Kubernetes DNS policies see [Pod DNS Policy](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy).</ul> |
```

Reply:
I'm breaking this out into a separate table in my next commit. I'm also including more information from Kubernetes documentation, or rather moving the information that was pulled out of Kubernetes in earlier content and associating it with the policy option on this dropdown list.